schema/schema.yaml

```diff
@@ -16,7 +16,8 @@ tables:
       datatype: string
       length: 30
       description: Publication reference identifier
-      ivoa:ucd: meta.id;meta.main
+      ivoa:ucd: meta.ref;meta.main
```

Comment:
Hmm, in this table, the reference is the identifier so I would use `meta.id`, in other tables it is the reference so for other tables it would be `meta.ref`